chore(grid): refactor export notes (#2835)
telerik/blazor-docs: components/grid/export/pdf.md

#### In This Article

  - [Basics](#basics)

>   - [How It Works](#how-it-works)

Reviewer: Note your actual heading. This link will not work.

}
````

## How the Export Works

> * For performance reasons, the PDF export mechanism draws each cell value on a single line. Any content that does not fit in the available space will be clipped. Text wrapping and PDF column resizing is not supported.

Reviewer: This one sounds more like a limitation.

> * Exporting to PDF in UI for Blazor is different from exporting in Kendo jQuery, where the full HTML is exported. The Blazor export to PDF will export the Grid to a table, similar to an Excel table. If you want [to export to PDF as HTML, you can use a custom approach](#custom-export).

Reviewer: I would reorder the sentences in here, as the essences is the second sentence, rather than the first one.

```suggestion
* The UI for Blazor PDF export will export the Grid to a table and will not export the full HTML. This behavior is different from when exporting in Kendo UI for jQuery. [To export to PDF as HTML, you can use a custom approach](#custom-export).
```

}
````

## How the Export Works

> The PDF export has the following specifics:
> 
> * The UI for Blazor PDF export will export the Grid to a table and will not export the full HTML. This behavior is different from when exporting in Kendo UI for jQuery. [To export to PDF as HTML, you can use a custom approach](#custom-export).

Reviewer 2: ```suggestion
The Grid exports its raw data to PDF and does not export the full HTML. This behavior is different from the PDF exporting in Kendo UI for jQuery. [To export to PDF as HTML, you can use a custom approach](#custom-export).
```

## Requirements

The PDF export has the following requirements:

* PDF export requires pixel widths for all columns. Widths in other units such as `%` or `em` cannot be translated correctly and the respective columns will collapse in the exported PDF file. The column widths for the PDF export can differ from the ones in the Grid configuration for the web browser. To set column widths for the PDF file only, use the `Width` property of the [`OnBeforeExportEventArgs.Columns`](slug:grid-export-events#for-pdf-export) members.

> * Provide appropriate `PaperSize` and `PageOrientation` properties. For example, if you want to render 20 columns (100px each) in an A4 sheet, then this will yield unexpected results. The column dimensions in a PDF file are fixed, thus they cannot be resized as in Excel, which requires the developer to ensure proper export dimensions.

Reviewer 2: I would mention / link this briefly in the basics section, in order to ensure that readers who experiment with PDF export for the first time and code while reading, will not hit something that they can perceive as a bug.
